mol-renderer/Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM debian:jessie
```

Comment:
Yeah, agreed, and I'd love to have suggestions.
I've certainly managed to strip things down enough that not too much should be needed. Still, the final image is 650MB... which is certainly more than required.